app/models/compute_resource.rb

```diff
@@ -123,6 +124,10 @@ def to_label
     "#{name} (#{provider_friendly_name})"
   end
 
+  def connection_options
+    http_proxy ? {:proxy => http_proxy.full_url} : {}
```

Comment:
If no proxy is set, this falls back to the proxy defined via environment variables. If you don't set a proxy via the UI, Foreman still uses a proxy. I find this confusing. What do you think to explicitly set this to use no proxy if it is not explicitly set?

Reply:
Just to complete this discussion: I think, we should do this in a separate PR if we want to do this.